tests/pr_test.go

```diff
@@ -145,6 +147,7 @@ func TestRunStandardSolutionSchematics(t *testing.T) {
 		{Name: "resource_group_name", Value: options.Prefix, DataType: "string"},
 		{Name: "region", Value: "us-east", DataType: "string"},
 		{Name: "deployment_name", Value: "da-mq-instance", DataType: "string"},
+		{Name: "provider_visibility", Value: "public", DataType: "string"},
```

Comment:
Why is this set to public? This is running in schematics, so it should use private. Otherwise we have no test running private (cc @maheshwarishikha @Aditya-ranjan-16 )

Reply:
oh, Sorry.
I missed it is running in Schematics.